src/main/java/org/dice_research/opal/civet/metrics/ProviderIdentityMetric.java

```diff
@@ -1,9 +1,6 @@
 package org.dice_research.opal.civet.metrics;
 
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.HashMap;
-
 import org.apache.jena.rdf.model.Model;
 import org.apache.jena.rdf.model.ModelFactory;
```

Comment:
There are some unused imports, please remove them.
According to codacy, it is showing all checks have passed but how it is showing all checks have passed, there should be some warnings or suggestions from codacy

Reply:
Unused imports removed.